FakebookNotifications/FakebookNotifications.DataAccess/NotificationsContext.cs

```diff
@@ -84,9 +84,9 @@ public bool ClearNotifications(IMongoCollection<User> userCol, IMongoCollection<
 
                 return true;
             }
-            catch
+            catch(Exception ex)
             {
-                _logger.LogWarning("Unable to clear seed data");
+                _logger.LogWarning("Unable to clear seed data" + ex);
```

Comment:
again, that's not it. this calls ToString which throws away most of the details of the exception.

there's overloads for every `Log*` method where the first parameter is a whole exception object.